.github/workflows/pr-build.yml

```diff
@@ -94,6 +94,12 @@ jobs:
         run : |
           ./test-galasactl-local.sh --buildTool gradle
 
+      - name: Turn script into an executable
+        run: chmod +x detect-secrets.sh
+    
+      - name: Run the detect secrets script
+        run: ./detect-secrets.sh
+
```

Comment:
Hi Ash, same comment as before on your Web UI PR - it will take a few minutes for the workflow to get to this point so can this be moved higher up so the author is notified as soon as possible of any secrets.